libtelemetry/src/lite/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -16,37 +17,49 @@
 public class MapboxTelemetry {
   private static final String NON_NULL_APPLICATION_CONTEXT_REQUIRED = "Non-null application context required.";
   private String accessToken;
-  private String userAgent;
   private TelemetryClient telemetryClient;
   private Callback httpCallback;
   private CopyOnWriteArraySet<TelemetryListener> telemetryListeners = null;
   private final CertificateBlacklist certificateBlacklist;
   private final ConfigurationClient configurationClient;
   static Context applicationContext = null;
 
+  @Deprecated
```

Comment:
Add note about why this is deprecated and what should be used instead of this deprecated constructor? Yes, pretty obvious that the new constructor added in this pr is now the one to use. Doesn't hurt to be thorough though.